packages/blocks/src/api/parser/test/index.js

```diff
@@ -107,46 +107,62 @@ describe( 'block parser', () => {
 			} );
 			expect( block ).toBeUndefined();
 		} );
-
-		it( 'should fallback to an older version of the block if the current one is invalid', () => {
-			registerBlockType( 'core/test-block', {
-				...defaultBlockSettings,
-				attributes: {
-					fruit: {
-						type: 'string',
-						source: 'text',
-						selector: 'div',
+		describe( 'fallback to an older version of the block if the current one is invalid', () => {
```

Comment:
When used as a verb, split the preposition:

```suggestion
		describe( 'fall back to an older version of the block if the current one is invalid', () => {
```